board/common/rootfs/usr/share/udhcpc/default.script

```diff
@@ -105,7 +105,13 @@ case "$ACTION" in
 	    /usr/sbin/avahi-autoipd -c $interface && /usr/sbin/avahi-autoipd -k $interface
 	fi
 
-	if /bin/ip addr add dev $interface $ip/$NETMASK $BROADCAST proto 5; then
+	if [ -n "$NETMASK" ]; then
+	    IP=$ip/$NETMASK
+	else
+	    IP=$ip
+	fi
+
+	if /bin/ip addr add dev $interface $IP $BROADCAST proto 5; then
```

Comment:
Do we have a ticket to fix symbolic protocol name lookups?

Reply:
You mean the `proto 5` ugliness? No, not that I'm aware of.

Reply:
New issue #285 tracked by new project.